hummingbot/market/hitbtc/hitbtc_api_order_book_data_source.py

```diff
@@ -0,0 +1,291 @@
+    async def get_active_exchange_markets(cls) -> pd.DataFrame:
+        """
+        Returned data frame should have symbol as index and include USDVolume, baseAsset and quoteAsset
+        """
+        async with aiohttp.ClientSession() as client:
+
+            markets_response, tickers_response = await safe_gather(
+                client.get(constants.REST_SYMBOL_URL),
+                client.get(constants.REST_TICKERS_URL)
+            )
+
+            markets_response: aiohttp.ClientResponse = markets_response
+            tickers_response: aiohttp.ClientResponse = tickers_response
+
+            if markets_response.status != 200:
+                raise IOError(
+                    f"Error fetching active {constants.EXCHANGE_NAME} markets information. " f"HTTP status is {markets_response.status}."
+                )
+            if tickers_response.status != 200:
+                raise IOError(
+                    f"Error fetching active {constants.EXCHANGE_NAME} tickers information. " f"HTTP status is {tickers_response.status}."
+                )
+
+            markets_data, tickers_data = await safe_gather(
+                markets_response.json(), tickers_response.json()
+            )
+
+            markets_data: Dict[str, Any] = {item["id"]: item for item in markets_data}
+            tickers_data: Dict[str, Any] = {item["symbol"]: item for item in tickers_data}
+
+            def merge(source, destination):
+                for key, value in source.items():
+                    if isinstance(value, dict):
+                        # get node or create one
+                        node = destination.setdefault(key, {})
+                        merge(value, node)
+                    else:
+                        destination[key] = value
+
+                return destination
+
+            data_union = merge(tickers_data, markets_data)
+
+            all_markets: pd.DataFrame = pd.DataFrame.from_records(data=list(data_union.values()), index="symbol")
+            all_markets.rename(
+                {"baseCurrency": "baseAsset", "quoteCurrency": "quoteAsset"}, axis="columns", inplace=True
+            )
+
+            btc_usd_price: float = float(all_markets.loc["BTCUSD"]["last"])
+            eth_usd_price: float = float(all_markets.loc["ETHUSD"]["last"])
+
+            usd_volume: List[float] = [
+                (
+                    volume * last if symbol.endswith(("USD", "USDT")) else
+                    volume * last * btc_usd_price if symbol.endswith("BTC") else
+                    volume * last * eth_usd_price if symbol.endswith("ETH") else
+                    volume
+                )
+                for symbol, volume, last in zip(all_markets.index,
+                                                all_markets.volume.astype("float"),
+                                                all_markets["last"].astype("float")
+                                                )
+            ]
```

Comment:
In closer inspection, HitBTC does have field called `volumeQuote`. This can be used to help you calculate the `USDVolume`. What you're doing will result in a less than accurate `USDVolume` value.
i.e.
For the trading pair `BTCUSD`
The `volumeQuote` provided is `390179053.2052016` while the `USDVolume` you've calculated is `3.88379e+08`.
You might want to revisit this